src/hash160.rs

```diff
@@ -28,7 +28,12 @@ use Error;
 
 /// Output of the Bitcoin HASH160 hash function
 #[derive(Copy, Clone, PartialEq, Eq, Default, PartialOrd, Ord, Hash)]
-pub struct Hash([u8; 20]);
+#[cfg_attr(feature="json", derive(schemars::JsonSchema))]
```

Comment:
nit: We use `feature="something"` and `feature = "something"` both. Could we add spaces here to eventually migrate to give all of them spaces?

Reply:
I'll fix this in my uses, but we don't consistently use either across the code as is.